### DISCUSSION TITLE
Export Nested Content

### DISCUSSION THREAD

**Discussion by CoenWarmer** [directus/directus#11517] [Feature Requests]
I have a collection model that has O2M or M2M fields with relations to other collections. Those collections can then have further O2M / M2M fields, and so on.

It would be awesome to have a functionality that can select an item and export that item as well as all related content items into JSON.

Then it can be used for backup, seeding or migrating to other Directus instances.

**Comment by azrikahar** [comment]
That would be awesome indeed. There's also past discussions on this such as #10448 and #6753. 👍

**Comment by rijkvanzanten** [comment]
For what it's worth, this is already possible when using the API directly (you can add the `export` query parameter to any request, including those with nested `fields` and other filters). We'll be adding a more advanced export flow into the app soon, to also allow this to be easily used from the app 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
